### PR TITLE
Move col pipe

### PR DESCRIPTION
**Pull request by scann0n**
## What this pull request accomplishes:

- Added functions to automagically create the MOVE `events` and `involved` materialized views via the `collisions_replicator_transfer.py` DAG
- Updated the DAG readme.md to reflect new procedures (aka the removal of the `dev` step

## Issue(s) this solves:

- #596

## What, in particular, needs to reviewed:

- sql scripts that run the new MOVE functions (there are 4; they are all in bdit_data-sources/collisions/, they're names are `func_refresh_ev_mv.sql`, `func_refresh_inv_mv.sql`, `func_refresh_norm_mv.sql` and `func_refresh_raw_fields_mv.sql`)
- the `collisions_replicator_transfer.py` DAG
- the updates to `bdit_data-sources\dags\readme.md`

## What needs to be done by a sys admin after this PR is merged

The DAG (`collisions_replicator_transfer.py`) is currently on my home folder, so it needs to be moved to `\data_scripts` and the symbolic link needs to be updated.

### CODE REVIEW

File: collisions/func_refresh_ev_mv.sql [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+CREATE OR REPLACE FUNCTION collisions_replicator.refresh_ev_mv()
+RETURNS void
+LANGUAGE 'sql'
+COST 100
+VOLATILE SECURITY DEFINER PARALLEL UNSAFE
+AS $BODY$
+    REFRESH MATERIALIZED VIEW CONCURRENTLY collisions_replicator.events WITH DATA;
+$BODY$;
+
+ALTER FUNCTION collisions_replicator.refresh_ev_mv()
```
Comment (chmnata) [inline collisions/func_refresh_ev_mv.sql:14]: maybe collisions_admin should own this?

File: collisions/func_refresh_ev_mv.sql [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+GRANT EXECUTE ON FUNCTION collisions_replicator.refresh_ev_mv() TO collisions_bot;
+
+GRANT EXECUTE ON FUNCTION collisions_replicator.refresh_ev_mv() TO scannon;
+
+REVOKE ALL ON FUNCTION collisions_replicator.refresh_ev_mv() FROM public;
+
+COMMENT ON FUNCTION collisions_replicator.refresh_ev_mv()
+IS '
+Refreshes collisions_replicator.events mat view with data. Called by a DAG at 3am daily.
```
Comment (chmnata) [inline collisions/func_refresh_ev_mv.sql:25]: Lets specific the DAG name as well :meow_salute:

File: collisions/func_refresh_inv_mv.sql [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+CREATE OR REPLACE FUNCTION collisions_replicator.refresh_inv_mv()
+RETURNS void
+LANGUAGE 'sql'
+COST 100
+VOLATILE SECURITY DEFINER PARALLEL UNSAFE
+AS $BODY$
+    REFRESH MATERIALIZED VIEW CONCURRENTLY collisions_replicator.involved WITH DATA;
+$BODY$;
+
+ALTER FUNCTION collisions_replicator.refresh_inv_mv()
```
Comment (chmnata) [inline collisions/func_refresh_inv_mv.sql:14]: owner to collisions_admin

File: collisions/func_refresh_norm_mv.sql [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+RETURNS void
+LANGUAGE 'sql'
+COST 100
+VOLATILE SECURITY DEFINER PARALLEL UNSAFE
+AS $BODY$
+    REFRESH MATERIALIZED VIEW CONCURRENTLY collisions_replicator.events_fields_norm WITH DATA;
+	REFRESH MATERIALIZED VIEW CONCURRENTLY collisions_replicator.involved_fields_norm WITH DATA;
+$BODY$;
+
+ALTER FUNCTION collisions_replicator.refresh_norm_mv()
```
Comment (chmnata) [inline collisions/func_refresh_norm_mv.sql:15]: owner to collisions_admin

File: collisions/func_refresh_raw_fields_mv.sql [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+RETURNS void
+LANGUAGE 'sql'
+COST 100
+VOLATILE SECURITY DEFINER PARALLEL UNSAFE
+AS $BODY$
+    REFRESH MATERIALIZED VIEW CONCURRENTLY collisions_replicator.events_fields_raw WITH DATA;
+	REFRESH MATERIALIZED VIEW CONCURRENTLY collisions_replicator.involved_fields_raw WITH DATA;
+$BODY$;
+
+ALTER FUNCTION collisions_replicator.refresh_raw_fields_mv()
```
Comment (chmnata) [inline collisions/func_refresh_raw_fields_mv.sql:15]: owner to collisions_admin

File: dags/collisions_replicator_transfer.py [hunk 6]
```diff
@@ -12,8 +12,12 @@
 #This script does things with those operators:
 #1) uses ACC to update acc_safe_copy
 #2) updates the mat view called collisions_no, which generates ids needed to...
-#3) update the events and involved mat views 
-#4) throws a sassy slack alert message when it fails
+#3) updates the events_cursed and involved_cursed mat views
```
Comment (chmnata) [inline dags/collisions_replicator_transfer.py:15]: We will need to update this DAG with Gabe's new slack username thingy

Comment (chmnata) [reply dags/collisions_replicator_transfer.py:15]: see this : https://github.com/CityofToronto/bdit_data-sources/pull/597
and this bug fix  https://github.com/CityofToronto/bdit_data-sources/pull/633/files PR for `name`

**Review by chmnata — CHANGES_REQUESTED** [review]
Looks good! The documentation for airflow is so nice :meow_wow: Just some permission stuff and we have to update how we call slack names with Gabe's new method. 

Side question, have the mat view's code been reviewed before?

**Comment by scann0n** [comment]
> Looks good! The documentation for airflow is so nice :meow_wow: Just some permission stuff and we have to update how we call slack names with Gabe's new method.
> 
> Side question, have the mat view's code been reviewed before?

Yes! This is MOVE's mat view code - I just copied and pasted it!

File: collisions/func_refresh_ev_mv.sql (hunk 2 above)
Comment (scann0n) [reply collisions/func_refresh_ev_mv.sql:25]: Implemented

File: collisions/func_refresh_norm_mv.sql (hunk 4 above)
Comment (scann0n) [reply collisions/func_refresh_norm_mv.sql:15]: Implemented!

File: dags/collisions_replicator_transfer.py (hunk 6 above)
Comment (scann0n) [reply dags/collisions_replicator_transfer.py:7]: Implemented!

**Comment by scann0n** [comment]
Okay I think that's all the changes for this request!

**Review by chmnata — APPROVED** [review]
Thanks @scann0n !!!

**Pull request merged**